internal/cachecontroller/cache_controller.go

```diff
@@ -287,5 +304,6 @@ func (c *InMemoryCacheController) invalidateIteratorCacheByObjectRelation(storeI
 // invalidateIteratorCacheByUserAndObjectType writes a new key to the cache.
 // An alternative implementation could delete invalid keys, but this approach is faster (see storagewrappers.findInCache).
 func (c *InMemoryCacheController) invalidateIteratorCacheByUserAndObjectType(storeID, user, objectType string, ts time.Time) {
+	c.logger.Debug("InMemoryCacheController invalidateIteratorCacheByUserAndObjectType invalidation")
```

Comment:
can probably rm or include similar debug log in other `invalidateIterator*` functions